src/Paprika.Tests/Store/AbandonedTests.cs

```diff
@@ -250,4 +243,61 @@ public async Task Reuse_in_grow_and_shrink()
             await block.Commit(CommitOptions.FlushDataAndRoot);
         }
     }
+
+    [Test]
+    [Category(Categories.LongRunning)]
+    public async Task Abandoned_chain_creation_with_overflow()
+    {
+        // Minimum iterations required to overflow the abandoned list (AbandonedList.MaxCount).
+        const int minIterations = 60;
+
+        // Minimum accounts required to overflow an abandoned page (AbandonedPage.Payload.MaxCount * 2).
+        const int numAccounts = 2000;
```

Comment:
Why not calculate the const on the basis of the comment here? Again, a change of implementation and consts should automatically change this test.